src/service/video-manager-impl.js

```diff
@@ -538,9 +580,13 @@ class VideoEntry {
       }
```

Comment:
I think visibility for the case where `autoplay` attribute is set but platfrom does not support it is ignored right now. Probably need to call `nonAutoplayLoadedVideoVisibilityChanged_` before returning in the `if` above.